server/webapp/WEB-INF/rails.new/app/assets/javascripts/console_log_foldable_section.js

```diff
@@ -64,37 +70,80 @@
       }
     }
 
-    function insertBasic(cursor, line) {
-      var output = c("dd", {class: "log-fs-line"}, ansi.ansi_to(line, formatter));
+    function lpad3(num) {
+      return ("00" + num).slice(-3);
+    }
+
+    // no, moment.js doesn't make this easy (at least not without some plugin)
+    function humanizeMilliseconds(duration) {
```

Comment:
Fair enough. It's probably worth adding the extra plugin for that sake. I'll do that.